packages/seed/src/cli.ts

```diff
@@ -197,6 +199,81 @@ function addRunCommand(cli: Argv) {
     );
 }
 
+function addPublishCommands(cli: Argv) {
+    cli.command("publish", "Publish releases", (yargs) => {
+        yargs
+            // TODO: Implement CLI releasing (currently npm publishing)
+            .command(
+                "generator <generator>",
+                "Publishes all latest versions of the generators to DockerHub unless otherwise specified. To filter to certain generators pass in the generator IDs as a positional, space delimited list.",
+                (yargs) =>
+                    yargs
+                        .positional("generator", {
+                            type: "string",
+                            demandOption: true,
+                            description: "Generator(s) to register"
+                        })
+                        // Version is a reserved option in yargs...
+                        .option("ver", {
+                            type: "string",
+                            demandOption: false
```

Comment:
should this be required?

Reply:
It's optional cause you can alternatively pass it paths to 2 versions files
e.g. `seed publish --generator python-sdk --versions-file file.yml --previous-versions-file file.yml` where it will run through and grab the latest version in the newer file that wasn't in the old file

It was easier to include in the command than to write a script that's run in actions

Reply:
The CLI then checks that at least one of `ver` or the files is passed in (that's that `.check` addition a little lower down)